IsomorphicString.java

```diff
@@ -0,0 +1,40 @@
+import java.io.*;
+import java.util.*;
+public class IsomorphicString {
+    static int size = 256;
+
+    public static void main(String[] args) {
+        boolean res = checkIsomorphic("aab", "xxy");
+        System.out.println(res);
+
+        res = checkIsomorphic("Egg", "Dad");
+        System.out.println(res);
+
+        res = checkIsomorphic("aab", "xyz");
+        System.out.println(res);
+
+    }
+
```

Comment:
Could you please add a description about the problem. Thank you!

Reply:
Given two strings s and t, determine if they are isomorphic.

Two strings are isomorphic if the characters in s can be replaced to get t.

All occurrences of a character must be replaced with another character while preserving the order of characters. No two characters may map to the same character but a character may map to itself.

For example,
Given "egg", "add", return true.

Given "foo", "bar", return false.

Given "paper", "title", return true.

Note:
You may assume both s and t have the same length.

Reply:
@Seralahthan Thank you! could you please update this info in the file with this a comment.

Reply:
@xaca Thank you! I have updated the file with the comment.